common/persistence/sql/sqlplugin/postgresql/admin.go

```diff
@@ -64,7 +64,7 @@ const (
 	// NOTE we have to use %v because somehow postgresql doesn't work with ? here
 	// It's a small bug in sqlx library
 	// TODO https://github.com/uber/cadence/issues/2893
-	createDatabaseQuery = "CREATE DATABASE %v"
+	createDatabaseQuery = `CREATE DATABASE "%v"`
```

Comment:
I believe using backticks to escape database name is the standard SQL.

```suggestion
	createDatabaseQuery = "CREATE DATABASE `%v`"
```

Reply:
Hi @rodrigozhou !
Sorry but it does not work. After applying your change on my computer, here is the error:
```
2023-08-29T21:45:11.754+0200    ERROR   Unable to create SQL database.  {"error": "pq: syntax error at or near \"`\"", "logging-call-at": "handler.go:94"}
exit status 1
```

Reply:
Here is the applied diff:

```diff
diff --git a/common/persistence/sql/sqlplugin/postgresql/admin.go b/common/persistence/sql/sqlplugin/postgresql/admin.go
index 2185e647a..3fd069f19 100644
--- a/common/persistence/sql/sqlplugin/postgresql/admin.go
+++ b/common/persistence/sql/sqlplugin/postgresql/admin.go
@@ -64,7 +64,7 @@ const (
        // NOTE we have to use %v because somehow postgresql doesn't work with ? here
        // It's a small bug in sqlx library
        // TODO https://github.com/uber/cadence/issues/2893
-       createDatabaseQuery = `CREATE DATABASE "%v"`
+       createDatabaseQuery = "CREATE DATABASE `%v`"
 
        dropDatabaseQuery = "DROP DATABASE IF EXISTS %v"
```